pkg/flowaggregator/flowaggregator.go

```diff
@@ -194,10 +196,11 @@ func NewFlowAggregator(
 	inactiveFlowRecTimeout time.Duration,
 	aggregatorTransportProtocol AggregatorTransportProtocol,
 	flowAggregatorAddress string,
+	producerInput *producer.ProducerInput,
```

Comment:
why we pass pointer instead of struct here? when creating kafka producer we only need the struct value right?

Reply:
Passing by value is a little slower than passing by reference considering the size of the produce input struct. 
https://github.com/antrea-io/antrea/pull/2516#discussion_r681316500